wolfcrypt/src/asn.c

```diff
@@ -11250,6 +11253,94 @@ static int wc_SetCert_LoadDer(Cert* cert, const byte* der, word32 derSz)
     return ret;
 }
 
+#ifdef WOLFSSL_ALT_NAMES
+
+/* AltName structure contains a sequence of names.
+   WC_ALTNAMESOFFSET is the max header size */
+#define WC_ALTNAMESOFFSET (MAX_SEQ_SZ) 
+
+int wc_CertAddAltName_ex(Cert *cert, byte *name, int nameSz, 
```

Comment:
Getting compile error:
```
wolfcrypt/src/asn.c:11262:5: error: no previous prototype for ‘wc_CertAddAltName_ex’ [-Werror=missing-prototypes]
 int wc_CertAddAltName_ex(Cert *cert, byte *name, int nameSz,
     ^~~~~~~~~~~~~~~~~~~~
```

Reply:
You must have a different compiler settings than me... This function is only intended to be called by the functions below it; would you be satisfied if it were declared "static"?  There are similar functions elsewhere in the build that do not have prototypes.